packages/amplify-util-uibuilder/src/__tests__/generateComponents.test.ts

```diff
@@ -32,18 +39,23 @@ jest.mock('../commands/utils/featureFlags', () => ({
   getTransformerVersion: jest.fn().mockReturnValue(2),
 }));
 
+const defaultStudioFeatureFlags = {
+  autoGenerateForms: 'true',
+  autoGenerateViews: 'true',
```

Comment:
do we need this anymore? Is views a part of feature flags?

Reply:
I'll create a follow-up task in our backlog to remove all traces of Views in our codebase.